### ISSUE TITLE
Enum from sqlalchemy is not correctly understood

### ISSUE THREAD

**Issue by marusak** [PyCQA/pylint#2161]
### Steps to reproduce
Create this file as `tst.py`
```
from sqlalchemy import Enum                                                                                  
p1 = Enum("A", "B", "C")                                                                                                          
print(p1.enums)
```
                                             
### Current behavior
`pylint-3 tst.py` returns
` E1101: Instance of 'A' has no 'enums' member (no-member)`
even though, it does have enums member.

### Expected behavior
No lint is reported.

### pylint --version output
```
pylint-3 2.0.0.dev
astroid 2.0.0.dev
Python 3.6.5 (default, Mar 29 2018, 18:20:46) 
[GCC 8.0.1 20180317 (Red Hat 8.0.1-0.19)]
```

**Comment by PCManticore** [comment]
I tried following the code in sqlalchemy to see where `enums` is defined, but this was somewhat of a wild goose chase as there was no obvious place where this happens: https://bitbucket.org/zzzeek/sqlalchemy/src/master/lib/sqlalchemy/sql/sqltypes.py. 
Feel free to ignore this error or use `generated-members` in order to provide the `enums` member or `ignored-classes` instead to ignore the `sqlalchemy.Enum` class.

**Comment by PCManticore** [comment]
Alternatively, a third solution would be to add an astroid brain tip here https://github.com/PyCQA/astroid/tree/master/astroid/brain.
